packages/tables/frontend/sirius-components-tables/src/rows/filters/RowFiltersMenu.tsx

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2025 CEA List.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import TableRowsIcon from '@mui/icons-material/TableRows';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Switch from '@mui/material/Switch';
+import Tooltip from '@mui/material/Tooltip';
+import { MouseEvent, useState } from 'react';
+import { RowFiltersButtonProps, RowFiltersMenuState } from './RowFiltersMenu.types';
+
+export const RowFiltersMenu = ({
+  readOnly,
+  rowFilters,
+  activeRowFilterIds,
+  onRowFiltersChange,
+}: RowFiltersButtonProps) => {
+  const initialState: RowFiltersMenuState = {
+    anchorEl: null,
+  };
+
+  const [state, setState] = useState<RowFiltersMenuState>(initialState);
+
+  const handleOpenRowFiltersMenu = (event: MouseEvent<HTMLElement>) => {
+    setState((prevState) => ({ ...prevState, anchorEl: event.currentTarget }));
+  };
+
+  const handleToggleFilter = (filterId: string) => {
+    let updatedActiveIds: string[] = [];
+    if (activeRowFilterIds.includes(filterId)) {
+      updatedActiveIds = activeRowFilterIds.filter((id) => id !== filterId);
+    } else {
+      updatedActiveIds = [...activeRowFilterIds, filterId];
+    }
+    onRowFiltersChange(updatedActiveIds);
+  };
+
+  return (
+    <>
+      {rowFilters.length > 0 && (
```

Comment:
Try not to use `&&` in such cases, while it's true that booleans will not be rendered by React, it can very quickly end up with surprising results since falsy or truthy statements can probably be rendered